runtime/bindings/python/tests/test_inference_engine/test_infer_request.py

```diff
@@ -6,136 +6,132 @@
 import pytest
 
 from tests.test_inference_engine.helpers import model_path, read_image
-from openvino import Core, Blob, TensorDesc, StatusCode
+from openvino import Core, Blob, TensorDesc, StatusCode, Tensor
```

Comment:
Are `Blob`, `TensorDesc` and `StatusCode` required here?

Reply:
Good point. Removed.